checkbox-ng/checkbox_ng/launcher/master.py

```diff
@@ -508,8 +509,16 @@ def finish_session(self):
         job_state_map = (
             self._sa.manager.default_device_context._state._job_state_map
         )
+        failing_outcomes = (
+            IJobResult.OUTCOME_FAIL,
+            IJobResult.OUTCOME_CRASH,
+            # Outcome may be still set to `None`. It may  happen if the job
+            # was interrupted or crashed hasn't been properly registered (yet)
+            None,
+        )
         self._has_anything_failed = any(
-            job.result.outcome == "fail" for job in job_state_map.values()
+            job.result.outcome in (None, "fail", "crashed")
```

Comment:
Did you mean to use `failing_outcomes` here? (I'm not sure, cause `IJobResult.OUTCOME_CRASH` is set to `crash`, not `crashed`, so I don't know if this is intentional)